### PR TITLE
feat: implement ruby cloud function to send sms

### PR DESCRIPTION
**Pull request by akashgp09**
Resolves: #92

 Implemented ruby cloud function to send SMS to other user's phone number.

 - [x]  Write code for the function.
 - [x]  Test the function using Appwrite.
 - [x]  Write a README.md explaining what function does, how to use and customize it.

IMG_20211015_000834

### CODE REVIEW

**Review by Meldiron — APPROVED** [review]
**Amazing** PR! 🤯  Approved by my, great job! Really clean PR, you just made my day.

**Comment by Meldiron** [comment]
@lohanidamodar Please review code.

File: ruby/send-sms/main.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# Send your message
+begin
+    @client = Twilio::REST::Client.new account_sid, auth_token
+    message = @client.messages.create(
+        body: message,
+        from: sender,
+        to: receiver
+        )
+    puts message
```
Comment (Meldiron) [inline ruby/send-sms/main.rb:23]: What exactly is an output? Can we show example output in README.md?

**Review by Meldiron — CHANGES_REQUESTED** [review]
One tiny comment after reviewing other PRs

File: ruby/send-sms/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+A sample Ruby Cloud function to send SMS to other user's phone number.
+
+## 📝 Environment Variables
+
+- **TWILIO_ACCOUNT_SID** - Twilio account SID
+- **TWILIO_AUTH_TOKEN** - Twilio auth token
+
+ℹ️ _Find your Account SID and Auth Token at twilio.com/console._
+## 🚀 Building and Packaging
```
Comment (lohanidamodar) [inline ruby/send-sms/README.md:11]:
```suggestion

## 🚀 Building and Packaging
```

File: ruby/send-sms/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+To package this example as a cloud function, follow these steps.
+
+```bash
+$ cd demos-for-functions/ruby/send-sms
+```
+
+You will first need to install the application's dependencies. You can do this manually using RubyGems, or you can install them using Bundler. To install via bundler, type:
+
+```bash
+$ bundle install
```
Comment (lohanidamodar) [inline ruby/send-sms/README.md:22]: Need to update the GEM_HOME to `.appwrite` sub folder and packaged with the function when deployed.

File: ruby/send-sms/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+```bash
+$ gem install twilio-ruby
+```
+
+- Ensure that your folder structure looks like this
+
+```
+.
+├── main.rb
```
Comment (lohanidamodar) [inline ruby/send-sms/README.md:35]:
```suggestion
├── main.rb
├── .appwrite/
```

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Looks great @akashgp09 just small changes!

File: ruby/send-sms/main.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# Import the helper libraries
+require 'twilio-ruby'
+require 'json'
+
+# Set the environment variables.
+account_sid = ENV['TWILIO_ACCOUNT_SID']
+auth_token = ENV['TWILIO_AUTH_TOKEN']
+
+# Get the sender's and receiver's mobile number and the message which has to be send.
+payload = JSON.parse(ENV['APPWRITE_FUNCTION_DATA'])
```
Comment (lohanidamodar) [inline ruby/send-sms/main.rb:10]: Hey can we handle some errors here, if data is not provided, not provided with all the correct fields, not with proper format? Thanks.

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
@akashgp09 Great work. Just a small request.

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**